tests/unit_tests/udf_wrapper_params/token_classification/multiple_model_multiple_batch_incomplete.py

```diff
@@ -54,60 +13,54 @@ class MultipleModelMultipleBatchIncomplete:
     batch_size = 3
     data_size = 2
     n_entities = 3
-    agg_strategy = "simple"
 
-    token_docid = 1
-    start = 0
-    end = 20
+    bfs_conn1, bfs_conn2 = make_number_of_strings(bucketfs_conn, 2) # todo why two in this test case? multiple model could still be same bfs con right?
```

Comment:
probably no specific reason might make sense to use the same bfsconn

Reply:
changed